core/EE_Load_Textdomain.core.php

```diff
@@ -42,13 +42,13 @@ public static function load_textdomain()
 
 
     /**
-     * The purpose of this method is to sideload the lang file for the given WPLANG locale (if necessary).
+     * The purpose of this method is to sideload all of the lang files for EE, this includes the POT file and also the PO/MO files for the given WPLANG locale (if necessary).
      *
      * @access private
      * @static
      * @return void
      */
-    private static function _maybe_get_langfile()
+    private static function _maybe_get_langfiles()
```

Comment:
same issue, can't rename public functions

Reply:
sigh ... oops... sry... this does not apply to this function since it is private... my bad